orm/encoding/ormfield/duration_test.go

```diff
@@ -114,14 +114,6 @@ func TestDurationOutOfRange(t *testing.T) {
 			},
 			expectErr: "seconds is out of range",
 		},
-		{
-			name: "positive seconds negative nanos",
-			dur: &durationpb.Duration{
-				Seconds: 0,
-				Nanos:   -1,
-			},
-			expectErr: "nanos is out of range",
-		},
```

Comment:
Do we need to remove this test case? Shouldn't validation fail here?

Reply:
I think that a duration with zero seconds and negative nanos is valid. This is from the documentation: 

```text
Durations less than one second are represented with a 0 seconds field and a positive or negative nanos field. For durations of one second or more, a non-zero value for the nanos field must be of the same sign as the seconds field. Must be from -999,999,999 to +999,999,999 inclusive.
```

Reply:
Oh right. It should be valid